.ruby-style.yml

```
@@ -864,7 +864,7 @@ Style/TrailingCommaInArrayLiteral:
 Style/TrailingCommaInHashLiteral:
   Description: 'Checks for trailing comma in hash literals.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas'
-  Enabled: false
+  Enabled: true
```

```suggestion
  Enabled: true
  EnforcedStyleForMultiline: no_comma
```

This is the Rubocop default enforced style and **doesn't** match https://github.com/everypolitician/everypolitician-data/blob/b63842260aafecfe6c67b1268a1e517d9b18d957/.rubocop_base.yml#L54. I think this will be more compatible with the Alaveteli code base.

---

👍 to `no_comma`.